pkg/api/types.go

```diff
@@ -256,6 +258,19 @@ type SecretVolumeSource struct {
 	Target ObjectReference `json:"target"`
 }
 
+// NFS represents and NFS Mount that lasts the lifetime of a pod
+type NFSVolumeSource struct {
+	// Server is the hostname or ip address of the NFS server
+	Server string `json:"server"`
+
+	// SourcePath is the exported NFS share
+	SourcePath string `json:"sourcePath"`
```

Comment:
I didn't want to use the term export or source. I'm fine with calling it path, I just wanted to make sure it was the path of the export.